saltgui/index.html

```diff
@@ -114,6 +133,7 @@ <h1>Manual Run</h1>
     </div>
 
     <script src='static/scripts/utils.js'></script>
+    <script src='static/scripts/stringify.js'></script>
```

Comment:
Would it  be an idea to add this function to the `output.js`file ?

Reply:
yes. but the functionality in `stringify.js` is not complete yet. It currently does not sort the dictionary keys, and it does not properly format the empty arrays and dictionaries that are embedded. That is on my backlog. When that functionality is complete I'll integrate the two when that is still reasonable. (i'll create a story for that)